File: Array/1252-Cells-with-Odd-Values-in-a-Matrix/1252-Cells-with-Odd-Values-in-a-Matrix.java
class Solution {
    public int oddCells(int m, int n, int[][] indices) {
        int[][] ans = new int[m][n];

        for(int[] mat : indices) {
            for(int i=0;i<n;i++) {
                ans[mat[0]][i]++;
            }

            for(int i=0;i<m;i++) {
                ans[i][mat[1]]++;
            }
        }
        int count = 0;
        for(int[] mat : ans) {
            for(int num : mat) {
                if(num % 2 != 0) count++;
            }
        }

        return count;
    }
}

File: Array/1252-Cells-with-Odd-Values-in-a-Matrix/1252-Cells-with-Odd-Values-in-a-Matrix.js
/**
 * @param {number} m
 * @param {number} n
 * @param {number[][]} indices
 * @return {number}
 */
var oddCells = function(m, n, indices) {
    let ans = [...Array(m)].map(e => [...Array(n)].fill(0));

    indices.map(mat => {
        const [x, y] = mat;

        for(let i=0;i<n;i++) {
            ans[x][i]++;
        }

        for(let i=0;i<m;i++) {
            ans[i][y]++;
        }
    })
    let count = 0;
    ans.map(mat => {
        mat.map(num => {
            if(num%2!=0) count++;
        })
    })

    return count;
};
